dependencies.yaml

```
@@ -486,6 +486,7 @@ dependencies:
         packages:
           - *numba
           - *numpy
+          - &tensordict tensordict>=0.1.2,<0.3.1
```

Typically good practice to disallow pre-releases, but it shouldn't matter for this case since there are no published pre-releases.
```suggestion
          - &tensordict tensordict>=0.1.2,<0.3.1a0
```